server/api/api.yaml

```
@@ -674,6 +674,60 @@ paths:
         500:
           $ref: "#/components/schemas/500"
 
+  /incidents/{id}/update-status:
+    post:
+      summary: Update an incident's status
+      operationId: updateStatus
+      security:
+        - BearerAuth: []
+      tags:
+        - Incidents
+      parameters:
+        - name: id
+          in: path
+          required: true
+          description: ID of the incident to update.
+          schema:
+            type: string
+            example: 1igmynxs77ywmcbwbsujzktter
+      requestBody:
+        description: Payload to change the incident's status.
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                status:
+                  type: string
+                  description: The new status. May be 'Reported', 'Active', 'Resolved', or 'Archived'
+                  example: Active
```

We can use the `enum` keyword here:
```suggestion
                status:
                  type: string
                  enum:
                    - Reported
                    - Active
                    - Resolved
                    - Archived
                  description: The new status.
                  example: Active
```

---

Awesome, thanks!